### PR TITLE
Update introduction.rst

### PR DESCRIPTION
**Pull request by martijn80**
the docs are missing important info that the var_dumper component needs to added as a bundle to the appkernel

### CODE REVIEW

File: components/var_dumper/introduction.rst [hunk 1]
```diff
@@ -17,6 +17,9 @@ You can install the component in 2 different ways:
 * :doc:`Install it via Composer </components/using_components>` (``symfony/var-dumper`` on `Packagist`_);
 * Use the official Git repository (https://github.com/symfony/var-dumper).
 
+.. note::
+Make sure the bundle is added in your ``app/AppKernel.php``
```
Comment (xabbuh) [inline components/var_dumper/introduction.rst]: This needs to be indented by four spaces.

**Comment by xabbuh** [comment]
Well, this section explains how to use the component outside of the full-stack framework. So there is no bundle to enable or something like that. Can you explain what issue you had that you think we should add something to the documentation? Maybe we can improve some other parts to make it easier to use the component.

**Comment by WouterJ** [comment]
@x

**Comment by xabbuh** [comment]
Oh, I didn't noticate that. Then we should fix that first imo.

**Comment by javiereguiluz** [comment]
I propose to replace this by #6347 which tweaks the note contents a bit. I think this note is important while we prepare the documentation of VarDumper + Symfony.

@martijn80 I've reused your original commit so you get credit for your work. Thanks for this contribution!

**Comment by xabbuh** [comment]
Closing here in favour of #6347. Thank you very much for starting this @martijn80!

**Pull request closed without merging**